pages/contact-us/[id].tsx

```diff
@@ -0,0 +1,218 @@
+import React, {
```

Comment:
don't need to import React from react as it is already imported by default by Next